_includes/articles/bitgo-musig2.md

```diff
@@ -0,0 +1,186 @@
+
+Now other MuSig2 implementers can also take advantage of the flexibility offered
+by allowing one of their signers not to manage state. By reviewing (and
+implementing) the draft specification during its development, we were able to
+contribute to the specification and be ready to launch MuSig2 signing soon after
+the specification was formally published as BIP327.
+
+## MuSig and PSBTs
+
+The [PSBT (Partially Signed Bitcoin Transaction)][topic psbt] format is intended to carry all
+information needed to sign a transaction between the parties (e.g., coordinator
+and signers in a simple case). The more information is needed for signing, the
+more valuable the format becomes. We examined the costs and benefits of
+expanding our existing API format with additional fields to facilitate MuSig2
+signing vs. converting to PSBT. We settled on converting to PSBT format for
+transaction data interchange, and it’s been a huge success. It’s not widely
+known yet, but BitGo wallets (except those using MuSig2, see the next paragraph)
+can now integrate with hardware signing devices that support PSBTs.
+
+There are not yet published PSBT fields for use in MuSig2 signing. For our
+implementation, we used proprietary fields that were based on a draft shared
+with us by [Sanket][]. This is one of the little talked about benefits of the PSBT
+format - the ability to include _whatever_ additional data might be needed for
+your custom transaction building or signing protocol in the same binary data
+format; with global, per-input, and per-output sections already defined. The
+PSBT specification separates the unsigned transaction from the scripts,
+signatures, and other data needed to eventually form a complete transaction.
+This separation can enable more efficient communication during the signing
+process. For example, our HSM can respond with a minimal PSBT including only its
+nonces or signatures, and they can be combined into the pre-signing PSBT easily.
+
+## Acknowledgements
+
+Thanks to Jonas Nick and Sanket Kanjalkar at Blockstream; Dawid Ciężarkiewicz at
+Fedi; and [Saravanan Mani][], David Kaplan, and the rest of the team at BitGo.
+
+{% include references.md %}
+[Brandon Black]: https://twitter.com/reardencode
+[BitGo]: https://www.bitgo.com/
+[MuSig paper]: https://eprint.iacr.org/2018/068
+[MuSig-DN]: https://eprint.iacr.org/2020/1057
+[MuSig2]: https://eprint.iacr.org/2020/1261
+[bitgo blog taproot]: https://blog.bitgo.com/taproot-support-for-bitgo-wallets-9ed97f412460
+[first tapscript multisig transaction]: https://mempool.space/tx/905ecdf95a84804b192f4dc221cfed4d77959b81ed66013a7e41a6e61e7ed530
+[bitgo blog musig2]: https://blog.bitgo.com/save-fees-with-musig2-at-bitgo-3248d690f573
+[FROST]: https://datatracker.ietf.org/doc/draft-irtf-cfrg-frost/
+[2-party multi-signatures]: https://duo.com/labs/tech-notes/2p-ecdsa-explained
+[RFC6979]: https://datatracker.ietf.org/doc/html/rfc6979
+[Dawid]: https://twitter.com/dpc_pw
```

Comment:
Did Dawid sign off on being mentioned? He sometimes seems to prefer getting less attention.

Reply:
Always happy to accept any praise. Unless there are any blame parts that I missed. I don't want to be blamed for anything.